openapi/resource-server.yaml

```diff
@@ -883,7 +883,7 @@ components:
           expiresAt: '2022-04-12T23:20:50.52Z'
           createdAt: '2022-03-12T23:20:50.52Z'
           updatedAt: '2022-03-12T23:20:50.52Z'
-      additionalProperties: false
+      unevaluatedProperties: false
```

Comment:
is there a way we can test this one?

Reply:
After some more testing, this is actually not doing anything when adding an additional property to the response. It does not cause any error. From reading [the underlying json-schema docs](https://json-schema.org/draft/2020-12/json-schema-core#name-unevaluatedproperties) and [other usages](https://github.com/search?q=path%3A*.yaml+OR+path%3A*.yml+%22unevaluatedProperties%22&type=code&p=2) it seems like this actually should go in the `incoming-payment-with-methods` (where the `allOf` is), but that didn't work either.